### DISCUSSION TITLE
In a Quarto Website: How to provide a .qmd template file to download?

### DISCUSSION THREAD

**Discussion by chaz-clark** [quarto-dev/quarto-cli#7848] [Q&A] [answered]
### Description

````qmd
_Use this [template](/Templates/ds250_project_template_clean.qmd) to submit your Client Report. The template has three sections (for additional details please see the [instructional template](/Templates/ds250_project_template.qmd)):_
````

The above code results in hyperlinks to `.html` rendered pages instead of being a downloadable link to the `.qmd` templates. Is there anyway to specify that these links remain `.qmd` files?

**Comment by mcanouil** [comment, marked as answer]
If you don't want the Quarto document to be rendered, you have to tell Quarto to not render it.
See <https://quarto.org/docs/websites/index.html#render-targets>.

```yaml
project:
  render:
    - "!/Templates"
```

**Reply by chaz-clark** [reply]
Thank you! per your code and the link i updated my file to contain the following code and it worked.

```
project:
  render:
    - "*.qmd”
    - "!/Templates”
```

**Reply by cscheid** [reply]
Just for everyone's clarity: the difference between

```
project:
  render:
    - "!/Templates"
```

and

```
project:
  render:
    - "*.qmd”
    - "!/Templates”
```

Is that the former requires 1.4 prerelease versions and the latter works with 1.3 and later.
